tablo/geom_utils.py

```diff
@@ -437,4 +438,51 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
```

Comment:
Food for thought: you may want to consider where the ESRI <-> WKT converters fit, relative to your new `wkt.py`  - could they go there instead?